MooseFS/docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+version: "3"
+
+services:
+  weba:
+    build:
+      context: ./weba
+    ports:
+      - "8001:5000"
+    volumes:
+      - type: volume
+        source: mfs_data
+        target: /var/lib/mfs
+    depends_on:
+      - chunkserver1
+      - chunkserver2
```

Comment:
Update dependency to just chunkserver 1. See my comments below for more details.